### PR TITLE
Bug 2025821: Make "Network Attachment Definitions" available to project-admin

### PR DESCRIPTION
**Pull request by avivtur**
**Fixes**:
https://bugzilla.redhat.com/show_bug.cgi?id=2025821

**Analysis / Root cause**:
redundant flag to check the access after this bug fix - https://bugzilla.redhat.com/show_bug.cgi?id=2021191 

**Solution Description**:
removing the redundant flag, and adding a missing `loadError` prop to the table component.

**Screen shots / Gifs for design review**:

**Before**:

before

**After**:

after

after-access-denied

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

**Comment by yaacov** [comment]
please remove the flag
no need to make it always true ...

File: frontend/packages/network-attachment-definition-plugin/src/plugin.ts [hunk 1]
```diff
@@ -42,6 +43,9 @@ const plugin: Plugin<ConsumedExtensions> = [
           './components/network-attachment-definitions/NetworkAttachmentDefinition' /* webpackChunkName: "network-attachment-definitions" */
         ).then((m) => m.NetworkAttachmentDefinitionsPage),
     },
+    flags: {
```
Comment (yaacov) [inline frontend/packages/network-attachment-definition-plugin/src/plugin.ts:46]: why ?

File: frontend/packages/network-attachment-definition-plugin/src/flags/check-nad-permissions.ts [hunk 2]
```diff
@@ -12,6 +13,7 @@ export const checkNadPermissions = async (setFeatureFlag: SetFeatureFlag) => {
       group: NetworkAttachmentDefinitionModel.apiGroup,
       resource: NetworkAttachmentDefinitionModel.plural,
       verb: 'list',
+      namespace: getActiveNamespace(),
```
Comment (yaacov) [inline frontend/packages/network-attachment-definition-plugin/src/flags/check-nad-permissions.ts:16]: does not make sense to me, the bug is about removing the unneeded check, not changing it.

Comment (avivtur) [reply frontend/packages/network-attachment-definition-plugin/src/flags/check-nad-permissions.ts]: thanks :) 
made some changes to the initial comment after the last iteration that should follow your tips

**Pull request merged**